examples/neural_networks/plot_random_neural_network.py

```diff
@@ -0,0 +1,155 @@
+"""
+========================================================================
+Impact of varying hyperparameters alpha and weight_scale on toy datasets
+========================================================================
+
+This illustrates how the random neural network behaves when varying the 
+regularization terms, weight_scale and alpha. Regularization usually
+affects the regularity or smoothness of the decision function.
+
+This example generates two plots, corresponding to each of the hyperparameters.
+They control regularization in that they constrain the coefficients of the
+decision function.
+
+If there is high bias - as can be indicated by a high training
+error, decreasing alpha or increasing weight_scale would decrease bias and 
+therefore reduce underfitting. Similarly, if there is high variance - as can be 
+indicated by a low training error, increasing alpha or decreasing weight_scale 
+would decrease variance and therefore reduce overfitting. Therefore, it is best
+to find a balance between bias and variance (or parameter values that are not 
+too high nor too low) by testing a range of values as seen in this example.
```

Comment:
Add a line "This can be achieved using cross-validation." or something like that.